source/common/request_info/utility.h

```diff
@@ -34,5 +34,17 @@ class ResponseFlagUtils {
   const static std::string RATE_LIMITED;
 };
 
+/**
+ * Utility class for RequestInfo.
+ */
+class Utility {
```

Comment:
Should this be a namespace instead of a class?

Reply:
I can change it. The rest of the code uses this pattern for the most part but I don't care either way if you have a strong preference.

Reply:
I don't feel strongly either way.